lib/operations/arithmetic/add.js

```diff
@@ -0,0 +1,12 @@
+function add() {
+  for (var _len = arguments.length, args = new Array(_len), _key = 0; _key < _len; _key++) {
+    args[_key] = arguments[_key];
+  }
+
+  return args.reduce(function (a, b) {
+    return parseFloat(a, 10) + parseFloat(b, 10);
+  }, 0);
+}
+
+add.code = '+';
```

Comment:
@lina-salih @mjenny 
Same here `code` as described in https://github.com/axa-ch/json-logic-js/pull/1/files#r259730014

`+` is an existing operator